docs/4.Integrations/pre-commit.md

```diff
@@ -70,6 +94,16 @@ repos:
         entry: checkov -d . --skip-check CKV_AWS_123
 ```
 
+When using the `diff` or `secrets` hooks, the last argument _must_ be `-f` due to how `checkov` and `pre-commit` interact:
+
+```yaml
+      - id: checkov_secrets_container
+        args:
+          - '--quiet'
+          - '-f' # required and must come last
+```
+
+
```

Comment:
```suggestion
By default, the container based pre-commit hooks use the `latest` tag. This can be overridden by declaring the version number in the entry field in the pre-commit config.

```yaml
    hooks:
      - id: checkov_container
        entry: bridgecrew/checkov:2.4.2 -d .
```

We should mention how to override in the docks.